modules/ROOT/pages/shared-resources.adoc

```diff
@@ -150,11 +150,29 @@ Example:
 
 === Referencing Shared Resources
 
-In the following example, `mule-domain-config.xml`, an HTTP connector is defined as a shared resource.
+In the following example, `mule-domain-config.xml`, an HTTP connector listener configuration is defined as a shared resource as well as an HTTP connector requester configuration.
 
 [source,xml,linenums]
 ----
-include::{examplesdir}/shared-resources_1.xml[]
+<?xml version="1.0" encoding="UTF-8"?>
+<mule-domain xmlns="http://www.mulesoft.org/schema/mule/domain"
+             xmlns:core="http://www.mulesoft.org/schema/mule/core"
+             xmlns:http="http://www.mulesoft.org/schema/mule/http"
+             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+             xsi:schemaLocation="
+               http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+                     http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+               http://www.mulesoft.org/schema/mule/domain http://www.mulesoft.org/schema/mule/domain/current/mule-domain.xsd">
+
+    <http:listener-config name="listenerConfig">
+        <http:listener-connection host="0.0.0.0" port="8080"/>
+    </http:listener-config>
+
+    <http:request-config name="domainLevel">
+        <http:request-connection host="localhost" port="9090"/>
+    </http:request-config>
+
+</mule-domain>
 ----
 
 Any Mule application associated with the domain can make use of the shared resource by referencing it within the configuration, just as you would reference a resource within the project itself. In the example below, the HTTP listener connector references the shared resource named `HTTP_Listener_Configuration`.
```

Comment:
Looks like `HTTP_Listener_Configuration` in the unedited paragraph should change to match the examples: `listenerConfig`